extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -1,26 +1,50 @@
-import {Layout, applyFillContent} from '#core/dom/layout';
+import {Layout} from '#core/dom/layout';
 
-const TAG = 'amp-story-shopping-config';
+import {Services} from '#service';
+
+import {devAssert} from '#utils/log';
+
+import {getRequestService} from '../../amp-story/1.0/amp-story-request-service';
+import {Action} from '../../amp-story/1.0/amp-story-store-service';
 
 export class AmpStoryShoppingConfig extends AMP.BaseElement {
   /** @param {!AmpElement} element */
   constructor(element) {
     super(element);
+    /** @private @const {?./amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+
+    /** @private @const {!./amp-story-request-service.AmpStoryRequestService} */
+    this.requestService_ = getRequestService(this.win, element);
```

Comment:
Please get the service through `Services`.
By doing this we will not need to import `getRequestService';`, resulting in a smaller bundle.

Please see [this example](https://github.com/ampproject/amphtml/blob/main/extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js#L259-L261)

The buildCallback will need to be revised to get this service.